Formula/gitg.rb

```diff
@@ -1,8 +1,8 @@
 class Gitg < Formula
   desc "GNOME GUI client to view git repositories"
   homepage "https://wiki.gnome.org/Apps/Gitg"
-  url "https://download.gnome.org/sources/gitg/3.32/gitg-3.32.1.tar.xz"
-  sha256 "24a4aabf8a42aa2e783e1fb5996ebb3c2a82a01b9689269f1329517ef124ef5a"
+  url "https://download.gnome.org/sources/gitg/41/gitg-41.tar.xz"
+  sha256 "7fb61b9fb10fbaa548d23d7065babd72ad63e621de55840c065ce6e3986c4629"
   revision 7
```

Comment:
```suggestion
```
`revision`s should be removed at version bumps.

Reply:
Ah ok